mailbox/store/src/main/java/org/apache/james/mailbox/store/event/distributed/RegisteredDelegatingMailboxListener.java

```diff
@@ -103,11 +104,14 @@ public void removeGlobalListener(MailboxListener listener, MailboxSession sessio
     }
 
     @Override
-    public void event(MailboxEvent event) {
+    public void event(Event event) {
         try {
             deliverEventToOnceGlobalListeners(event);
             deliverToMailboxPathRegisteredListeners(event);
-            sendToRemoteJames(event);
+            if (event instanceof MailboxEvent) {
```

Comment:
why don't you propagate events in the same way than mailboxevents?

Reply:
Very simple:
 - We don't want to handle serialization for now of these events
 - They are used only in global listener for now so don't need to be sent to other nodes (are GlobalEvents are managed locally)

With @aduprat  we decided to save time....